ingestion/src/metadata/ingestion/ometa/utils.py

```diff
@@ -23,6 +24,8 @@
 
 T = TypeVar("T", bound=BaseModel)  # pylint: disable=invalid-name
 
+logger: logging.Logger = logging.getLogger(__name__)
```

Comment:
We could also move the timer decorator to is own module and init there a logger with
`logger: logging.Logger = logging.getLogger("Timer")`, which might be helpful to locate the logs